source/docs/advanced/payments/adyen.md

```diff
@@ -5,46 +5,67 @@ title: Adyen integration
 
 This page contains information about the different ways you can accept payments with [Adyen](https://www.adyen.com/) in your Front-Commerce application.
 
-## Magento2 module
+## As a Front-Commerce payment (no Magento modules needed)
```

Comment:
I don't think it is necessary to add the `(no Magento modules needed)` also please be aware that when Algolia crawls the docs, it reads secondary headers as well, which means that this would be indexed.

Reply:
renamed and added a note at the begining:
```md
P.S. No Magento modules are needed if you use this method. If you however want to integrate with the adyen Magento module please refer to [Magento2 Integration section](#As-a-Magento2-Integration).
```